tinygo/snippets/poll_oneoff/main.go

```diff
@@ -18,8 +18,7 @@ var pollWasm []byte
 
 func main() {
 	ctx := context.Background()
-	ctx = context.WithValue(ctx, experimental.FunctionListenerFactoryKey{},
-		logging.NewHostLoggingListenerFactory(os.Stderr, logging.LogScopeFilesystem|logging.LogScopePoll|logging.LogScopeSock))
+	ctx = experimental.WithFunctionListenerFactory(ctx, logging.NewHostLoggingListenerFactory(os.Stderr, logging.LogScopeFilesystem|logging.LogScopePoll|logging.LogScopeSock))
```

Comment:
`wazero` has (breakingly 😞) changed how they allow user to enable debug logging.